src/dwarf.cpp

```diff
@@ -341,16 +341,15 @@ void line_header::read(freader& s, bool needs_byteswap) {
 }
 
 /**************************************************************************************************/
-
-std::size_t fatal_attribute_hash(const attribute_sequence& attributes) {
-#if ORC_FEATURE(PROFILE_DIE_DETAILS)
-    ZoneScoped;
-#endif // ORC_FEATURE(PROFILE_DIE_DETAILS)
-
-    // We only hash the attributes that could contribute to an ODRV. We also sort that set of
-    // attributes by name to make sure the hashing is consistent regardless of attribute order.
-    constexpr const std::size_t max_names_k{32};
-    std::array<dw::at, max_names_k> names{dw::at::none};
+// It is fixed to keep allocations from happening.
+constexpr const std::size_t max_names_k{32};
```

Comment:
nit pick but fascinating: is there meaning to a `const constexpr`? And did you mean `static constexpr`?

Reply:
`constexpr const` is redundant, yeah. I have removed the `const`.

And great question about `static constexpr`. I looked into this a bit, and it would make sense to add the `static` qualification if the `constexpr` expression needs a runtime instance (e.g., to take its address when passing it to a function.) The `static` in that case ensures only one such instance is created (and in a threadsafe way). Given that `max_names_k` is not used like that, just `constexpr` should suffice (though I don't think there's any harm in adding the `static`).